code/modules/projectiles/ammo_datums.dm

```diff
@@ -1533,15 +1535,29 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	if(isclosedturf(T))
 		T = get_turf(get_step(T, turn(P.dir, 180))) //If the turf is closed, we instead drop in the location just prior to the turf
 
-	drop_nade(T)
+	drop_nade(T, puddle_duration)
 
 /datum/ammo/xeno/acid/heavy/do_at_max_range(obj/projectile/P)
-	drop_nade(get_turf(P))
+	drop_nade(get_turf(P), puddle_duration)
+
 
-/datum/ammo/xeno/acid/drop_nade(turf/T) //Leaves behind a short lived acid pool; lasts for 1-3 seconds.
+/datum/ammo/xeno/acid/drop_nade(turf/T, duration = 1 SECONDS) //Leaves behind an acid pool; defaults to 1-3 seconds.
 	if(T.density)
 		return
-	new /obj/effect/xenomorph/spray(T, 10)
+	new /obj/effect/xenomorph/spray(T, duration)
+
+
+///For the Spitter's Scatterspit ability
+/datum/ammo/xeno/acid/heavy/scatter
+	damage = 15
+	flags_ammo_behavior = AMMO_XENO|AMMO_EXPLOSIVE|AMMO_SKIPS_ALIENS
+	bonus_projectiles_type = /datum/ammo/xeno/acid/heavy/scatter/bonus
+	bonus_projectiles_amount = 5
+	bonus_projectiles_scatter = 10
+	puddle_duration = 2 SECONDS //Lasts 2-4 seconds
```

Comment:
Why is this acid/heavy/scatter rather than /acid/scatter?

Reply:
so it inherits the puddle code and doesn't have to duplicate that